Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -105,31 +105,34 @@ public class BellerophonManager: NSObject {
         }
     }
 
-    private func stopTimer() {
+    internal func stopTimer() {
         retryTimer?.invalidate()
         retryTimer = nil
     }
 
-    private func performForceUpdate() {
+    internal func performForceUpdate() {
         delegate?.shouldForceUpdate()
     }
 
-    private func displayKillSwitch() {
+    internal func displayKillSwitch() {
         if !killSwitchWindow.keyWindow {
             killSwitchView.frame = killSwitchWindow.bounds
             delegate?.bellerophonWillEngage?(self)
-            killSwitchWindow.makeKeyAndVisible()
+            // if there's no such keyWindow in the application, do not show killSwitchWindow
```

Comment:
I did not understand this part. When will this situation happen and why don't we want to show the KS window?